### ISSUE TITLE
Alternate spelling alphabets

### ISSUE THREAD

**Issue by DonaldTsang** [korn101/PiNumberStation#13]
I would recommend creating a variation of the alphabet to differentiate upper case, lower case and special characters for base64, base58 and other encodings for use in IPFS multihash based applications.
We can get some ideas in https://en.wikipedia.org/wiki/Spelling_alphabet (if the word is 2 syllables it is upper case, if it is 3 syllables it is lower case)
@hiryu85 might want to take a look.

**Comment by DonaldTsang** [comment]
A list from Wikipedia:
- A Able Ace Ack Actor Affirm Alfa Alpha Apple Ash Authority
    - Name: Adam Adams Alice Ana Andrew
    - Places: America Amsterdam Argentine
- B Back Baker Beer Beta Bills Boy Bravo Brother Butter
    - Name: Benjamin Bertha
    - Places: Baltimore Boston Brussels
- C Canteen Capture Cast Chain Coca Coco Cork
    - Name: Charles Charlie
    - Places: Canada Casablanca Chicago Chile
- D Dado Delta Destroy Diver Dock Dog
    - Name: David Dora Duff
    - Places: Damascus Denmark Denver Don Dover
- E Eagle Eastern Easy Echo Egg Englishmen Esses Extra
    - Name: Eddy Eddystone Edison Edward Elsa Emma Equis
    - Places: Ecuador Egypt England
- F Father Fiesta Fisher Fox Foxtrot Fractious
    - Name: Frank Freddie Freddy Frederick
    - Places: Finland Florida France Francisco
- G Galloping Gangway Gig Gold Golf Gramma 
    - Name: George
    - Places: Gallipoli Geneva Germany Gibraltar Greece
- H Halliard Have High Hombre Horse Hotel How Hypo
    - Name: Harry Henry
    - Places: Hanover Havana Hawaii Honolulu
- I Ice Ida Ink Insect Int Inter Interrogatory Invariably Item
    - Name: Isaac
    - Places: India Ireland Italia Italy
- J Jig Jockey Jug Juggling Jupiter
    - Name: Jack Jake James John Johnny Jude Juliett Julietta Julio
    - Places: Japan Jersey Jerusalem
- K Kilo Kilogramme Kilowatt King Knapsack Knights
    - Name: Kate Kimberley
    - Places: Kentucky Khartoum
- L Lash Liter Loose Love Lugger 
    - Name: Lewis Lincoln Louis Lucy Luis
    - Places: Lima Liverpool London Luxembourg
- M Mama Managing Metro Monkey Mother Mule Musket 
    - Name: Mary Mike Maestro
    - Places: Madagascar Madrid Mexico Montreal
- N Nab Nan Nectar Negat Neptune Net Never New Nickel North November Nuts
    - Name: Nancy Nellie Nelly Nora Norma
    - Places: Neufchatel New-York Newark Nicaragua Norway
- O Oak Oboe Ocean October Opal Opera Option Orange Owners Oyster
    - Name: Oliver Oscar Otto
    - Places: Oble Ontario Oslo Ostend
- P Page Papa Pip Pistol Play Polka Prep Pudding Pup
    - Name: Paul Peter
    - Places: Pacific Paris Peru Portugal
- Q Quack Quad Quadrant Quail Queen Queenie 
    - Name: Quaker
    - Places: Quebec
- R Radio Raft Reefer Remarks Roger Rot Rush
    - Name: Robert Romeo Ropert
    - Places: Rivoli Roma Romania Rome Rosa
- S Sail Santa Scout Shipmate Sugar Support
    - Name: Sam Samuel Sara Sierra Susan
    - Places: Santiago Sardinia Sweden
- T Talk Tango Tare The Tide Toc Topsail 
    - Name: Thomas Tom Tommy
    - Places: Texas Tokyo Tripoli
- U Uncle Uniform Union Unit United Unless Unload Use 
    - Name: Ursula
    - Places: Upsala Uruguay Utah
- V Vast Vessel Vice Victory Vindictive Vinegar Violet
    - Name: Vic Victor Victoria
    - Places: Valencia Venezuela
- W Watch Wednesday When Whiskey Winch Windage
    - Name: William Willie
    - Places: Washington
- X X-Ray X-Mas Xpeditiously
    - Name: Xanthippe Xerxes
    - Places: Xaintrie
- Y Yacht Yankee Yellow Yeoman Yoke Young Your
    - Name: Yankey Yolanda
    - Places: Yale Yokohama York Yorker
- Z Zebra Zed Zero Zigzag Zoo 
    - Name:
    - Places: Zanzibar Zulu Zululand Zurich

https://hackmd.io/9Ur77kvuRCC3q9jcwuq4OQ

P.S. if you really want binary try https://en.wikipedia.org/wiki/PGP_word_list or https://github.com/singpolyma/mnemonicode

**Comment by virtadpt** [comment]
+1 for the PGP Wordlist.

**Comment by DonaldTsang** [comment]
@virtadpt agreed but we should also have a wordlist for base64 as it is pretty useful in a sci-fi setting

**Comment by virtadpt** [comment]
Oh, definitely.  I'm not saying "or," I'm saying "in addition to."

**Comment by DonaldTsang** [comment]
@virtadpt cleaned the list to be more organized, perhaps we can make it such that the base64 code can be well designed?
